app/domain/commands/credentials/change_password.rb

```diff
@@ -6,12 +6,18 @@ module Commands
   module Credentials
 
     ChangePassword ||= CommandClass.new(
-      dependencies: {},
+      dependencies: {
+        audit_log: ::Audit.logger
+      },
```

Comment:
Ok so this somewhat changes my comment on previous commit, but it remains the case that one of our dependencies is still sneaking in through the input.  

My recommendation might change now to:

1. Keep the command class.
2. Add a comment stating the problem above dependencies and inputs and link to issue.